.gitignore

```diff
@@ -138,3 +138,6 @@ dmypy.json
 !.vscode/launch.json
 !.vscode/extensions.json
 *.code-workspace
+
+# test XML files
+*.xml
```

Comment:
This excludes now all the XML files from the project, not only test XML files. If the purpose is to exclude only test XML files, I recommend using a specific folder that is excluded or adds some prefix to the test files, then it does not have an effect on other XML files later if there become needs for these.